### ISSUE TITLE
Is the project dead?

### ISSUE THREAD

**Issue by UserGit-2021** [N1GHT-MAR3/JuicedEditor#1]
Latest release of 2022.

**Comment by N1GHT-MAR3** [comment]
No. I've just been busy with life and battling a funk for a while. There've been a few features added to the repo since version 1 - I just haven't finished enough to make a full release yet.

Version 2 (and beyond) will come sooner rather than later, I promise.

**Comment by UserGit-2021** [comment]
> No. I've just been busy with life and battling a funk for a while. There've been a few features added to the repo since version 1 - I just haven't finished enough to make a full release yet.
> 
> Version 2 (and beyond) will come sooner rather than later, I promise.

Thanks for the answer. I just wanted to know if it was worth the wait.

**Comment by N1GHT-MAR3** [comment]
Reopening issue until v2 is released.
